README.md

```diff
@@ -0,0 +1,52 @@
+
+                HacktoberFest Starter Project
+
+Use this project to make your first contribution to an open source project on GitHub. Practice making your first pull request to a public repository before doing the real thing!
+
+Celebrate Hacktoberfest by getting involved in the open source community by completing some simple tasks in this project.
+
+This repository is open to all members of the GitHub community. Any member may contribute to this project without being a collaborator.
+
+https://github.com/DotSlashCommunity/hacktoberfest
+
+
+	What is Hacktoberfest?
+
+
+A month-long celebration from October 1st - 31st sponsored by Digital Ocean and GitHub to get people involved in Open Source. Create your very first pull request to any public repository on GitHub and contribute to the open source developer community.
+
+https://hacktoberfest.digitalocean.com/
+
+How to contribute to this project
+
+Here are 2 quick and painless ways to contribute to this project:
+
+    Add your name to the ksdme.md file
+    Put your profile links in ksdme.md file
+    
+Choose one or 2, make a pull request for your work and wait for it to be merged!
+
+	Getting started
+
+Fork this repository (Click the Fork button in the top right of this page, click your Profile Image)
+
+Clone your fork down to your local machine
+
+git clone https://github.com/your-username/hacktoberfest.git
```

Comment:
Please enclose code/command fragments in '```', for example:
```
git clone https://github.com/your-username/hacktoberfest.git
```
Please read https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code.